aws_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+#
+
+DB_ENDPOINT=`aws ssm get-parameters --names db_endpointdev --region us-east-1 | jq -r '.Parameters[0].Value'`
+ECS_LB=`aws ssm get-parameters --names ecs_lb --region us-east-1 | jq -r '.Parameters[0].Value'`
+EC2_IP=`aws ssm get-parameters --names ec2_ip_dev --region us-east-1 | jq -r '.Parameters[0].Value'`
+RDS_PW=`aws ssm get-parameters --names rds_pw --region us-east-1 | jq -r '.Parameters[0].Value'`
+
+
+
+
+mv .env.example .env
```

Comment:
We should either create a .env that is better suited for a production deployment or just pull the environment variables that are injected into the container by SSM. It looks like this should be possible with the `secrets` key in CloudFormation? https://docs.aws.amazon.com/AmazonECS/latest/developerguide/secrets-envvar-ssm-paramstore.html. We don't need the `.env` - if the secrets are in the container then we can just read with with `django-environ`. Otherwise, the `example.env` needs to be copied and updated so nothing in there could be a security risk. Namely `DJANGO_SUPERUSER` vars, `ELASTIC` and `KIBANA` passwords, and `SECRET_KEY`.